Shared/Coordinators/Navigation/NavigationRoute/NavigationRoute+Item.swift

```diff
@@ -244,9 +244,18 @@ extension NavigationRoute {
             )
         }
     }
-
     #endif
 
+    static func itemMetadataRefresh(viewModel: RefreshMetadataViewModel) -> NavigationRoute {
+        NavigationRoute(
+            id: "itemMetadataRefresh",
+            style: .sheet
+        ) {
+            ItemMetadataRefreshView(viewModel: viewModel)
+                .presentationDetents([.medium, .large])
```

Comment:
I don't want to think about introducing additional detents quite yet, let's keep it as it would naturally be presented for now.